data/convert_replays.py

```diff
@@ -110,41 +129,37 @@ def get_structures(proto_game, gdf: pd.DataFrame):
     # Init structures from json
     goal_seconds, goal_frames, goal_scorers, goal_teams = [], [], [], []
     # Player Data
-    z_team_names, o_team_names = [], []
+    # ~1 in 3 games have the blue team as team "1" instead of "0". We have to fix the team order in these cases.
```

Comment:
blue team in index 1?

Reply:
Yes. In the prior method, the teams would be wrong but goals would be "correct" although not matching. So goals in index 0 were always blue teams goals but blue team would sometimes be index 1. The new way has the goals matching the teams, for better or for worse.